cesium/internal/testdata/v1/db-data/2/1.domain

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Interesting. Why does this file have a blank line at the start?

Reply:
Seems like all binary files have a blank line at the start, will investigate